_studio/shared/include/mfx_utils.h

```diff
@@ -99,9 +99,9 @@ mfxU64 GetMfxTimeStamp(mfxF64 ts)
 }
 
 inline
-bool LumaIsNull(mfxFrameSurface1 * surf)
+bool LumaIsNull(const mfxFrameSurface1 * surf)
```

Comment:
This function implementation is only for Y410. It should be named and used accordingly.
Otherwise check .Y or .Y410 or whatever depending on FOURCC.

Reply:
Fixed